metricbeat/module/zookeeper/mntr/mntr.go

```diff
@@ -0,0 +1,41 @@
+/*
+
+Implementing mntr call from https://zookeeper.apache.org/doc/trunk/zookeeperAdmin.html
+
+Tested with Zookeeper 3.4.8
+
+*/
+package mntr
+
+import (
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+	"github.com/elastic/beats/metricbeat/helper"
+	"github.com/elastic/beats/metricbeat/module/zookeeper"
+)
+
+func init() {
+	helper.Registry.AddMetricSeter("zookeeper", "mntr", New)
+}
+
+func New() helper.MetricSeter {
+	return &MetricSeter{}
+}
+
+type MetricSeter struct {
+}
+
+func (m *MetricSeter) Setup(ms *helper.MetricSet) error {
+	return nil
+}
+
+func (m *MetricSeter) Fetch(ms *helper.MetricSet, host string) (event common.MapStr, err error) {
+	command := "mntr"
```

Comment:
This looks like a constant so move its declaration out of the method and into a `const` block.